packages/omg-js-rootchain/src/index.js

```diff
@@ -39,7 +39,7 @@ class RootChain {
   }
 
    /**
-   * deposit Token to rootchain
+   * deposit Token to rootchain (caller must be msg.sender)
```

Comment:
Not sure what this comment addition means... `msg.sender` doesn't really make sense in this context.
Maybe 'caller must be the token owner'?

Reply:
ah yes, that's what I was going for. We are making it explicit that the caller of the function is the token owner